### ISSUE TITLE
iTunes 8 Incompatibility

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MziRintu/iphonebrowser#53]
```
What steps will reproduce the problem?
1. Install iTunes 8
2. Run Program
3. Instant Crash (Send error report)

What is the expected output? What do you see instead?
Crash


What version of the product are you using? On what operating system?
1.7

Please provide any additional information below.


```

Original issue reported on code.google.com by `spira1he...@gmail.com` on 10 Sep 2008 at 1:56

**Comment by GoogleCodeExporter** [comment]
```
Same problem
after unistalling iTunes problem persists

Descrição:
  Stopped working

Assinatura do problema:
  Nome do Evento de Problema:   APPCRASH
  Application Name: iPhoneBrowser.exe
  Application Version:  1.7.0.0
  Application Timestamp:    48912d88
  Fault Module Name:    ntdll.dll
  Fault Module Version: 6.0.6001.18000
  Fault Module Timestamp:   4791a7a6
  Exception Code:   c0000005
  Exception Offset: 00043387
  Versão do sistema operacional:   6.0.6001.2.1.0.256.1
  Identificação da Localidade:    1046


```

Original comment by `fabiop...@gmail.com` on 11 Sep 2008 at 5:27

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.80
```

Original comment by `NetMage....@gmail.com` on 11 Sep 2008 at 6:08
- Changed state: **Fixed**
- Added labels: **Priority-Critical**
- Removed labels: **Priority-Medium**
